x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/action_connectors/import_rule_action_connectors.test.ts

```diff
@@ -178,6 +178,7 @@ describe('checkRuleExceptionReferences', () => {
               '1 connector is missing. Connector id missing is: cabc78e0-9031-11ed-b076-53cc4d57aaf1',
             status_code: 404,
           },
+          id: 'cabc78e0-9031-11ed-b076-53cc4d57aaf1',
```

Comment:
Interesting, you provide both `id` and `rule_id` in tests while it should be one of them according to the operated types. See my question regarding them above.

Reply:
They are different `id` is the actionConnectors ids and the `rule_id` is the rule to be imported.

And we need it, in case the user has multiple rules in one file and not all of them are failing

Reply:
I see. For some reason `actionsImporter` returns a comma separated list of connector identifiers in `id` field and we don't have any breakdown on this. So this list corresponds to a rule.